docs/contributing-to-dev.md

```diff
@@ -11,7 +11,13 @@ We accept contributions via Pull Requests on [Github](https://github.com/Laravel
 * fork this repository
 * create a directory in *src* with the short name of the language (ex: fr for French) from ISO-639-1 (
   see [Wikipedia](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) )
-* copy these four files of the English version with your translation : from `script/en` directory
+* copy these five files of the English version with your translation : from `script/en` directory
+    * `auth.php`
+    * `pagination.php`
+    * `passwords.php`
+    * `validation.php`
+    * `validation-inline.php`
+* Keep in mind that the `validation-online.php` file does not come with Laravel and the idea of ​​this file is not to put a specific name to each attribute (as in `validation.php`) but a generic name for the validation attributes. Therefore in the translations of this file the placeholder `:attribute` **should not** appear.
```

Comment:
validation-online.php => validation-inline.php

Reply:
Excuse me for typo. Fixed!

Reply:
validation-offline 😀